### PR TITLE
removed option to specify github teams for "needs-sig" label

### PR DESCRIPTION
**Pull request by AradhanaSingh**
when needs-sig label is applied, two options are available. The first option, github teams need to be specified and for the second one, sig label needs to be specified manually. This PR removes the option of adding github teams(option 1) from the usage of needs-sig label.
#13587

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -426,16 +426,10 @@ require_matching_label:
...
 
-    1. mentioning a sig: `@kubernetes/sig-<group-name>-<group-suffix>`
-        e.g., `@kubernetes/sig-contributor-experience-<group-suffix>` to notify the contributor experience sig, OR
-
-    2. specifying the label manually: `/sig <group-name>`
-        e.g., `/sig scalability` to apply the `sig/scalability` label
-
-    Note: Method 1 will trigger an email to the group. See the [group list](https://git.k8s.io/community/sig-list.md).
-    The `<group-suffix>` in method 1 has to be replaced with one of these: _**bugs, feature-requests, pr-reviews, test-failures, proposals**_.
+    specifying the label manually: `/sig <group-name>`
```
Comment (cblecker) [inline config/prow/plugins.yaml]: Thanks for doing this! I'd like to reword this a bit, as if we are only offering one option, there isn't anything that's more "manual" than the other.

Maybe something like:
```
There are no sig labels on this issue. Please add an appropriate label by using one of the the following commands:
- `/sig <group-name>`
- `/wg <group-name>`
- `/committee <group-name>`

Please see the [group list](https://git.k8s.io/community/sig-list.md) for a listing of the SIGs, working groups, and committees available.
```

**Comment by idealhack** [comment]
@AradhanaSingh Hi, thank you for work on this! could you please address the review feedback?

File: config/prow/plugins.yaml (hunk 1 above)
Comment (AradhanaSingh) [reply config/prow/plugins.yaml]: done, please review

**Comment by idealhack** [comment]
/lgtm

either you can squash the commits, or this will need a squash label

**Comment by AradhanaSingh** [comment]
/tide/merge-method-squash

**Comment by idealhack** [comment]
@AradhanaSingh that label can be added manually, let's wait for @cblecker :)

**Pull request merged**